adapters/wazirx-nft-Xauction-bsc/index.js

```diff
@@ -0,0 +1,162 @@
+require("dotenv").config();
+
+const web3 = require("web3");
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const BSC = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const TRANSFER_TOPIC = "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const { AVAILABLE_PROTOCOLS } = require("../../sdk/constants");
+
+const WRX_TOKEN = "0x8e17ed70334c87ece574c9d537bc153d8609e2a3";
+const BNB_TOKEN = "bnb";
+
+class WazirxNft {
+    constructor() {
+        this.name = "WazirXNFT";
+        this.symbol = "BNB";
+        this.token = BNB_TOKEN;
```

Comment:
I don't think that you are always using BNB for payment. https://bscscan.com/tx/0x070ecb620619fea1ab9ed37442f5a81c85d6079fe638a27b1a29c21cad2148df this sale for e.g. is using WRX token for payment.

Reply:
Yes. we are using WRX as well for payments, while returning sale objects we are converting WRX to BNB to maintain uniformity.

Reply:
Please return price in `WRX` in response. We are converting price to `BNB` when needed under the hood. We want to keep data as accurate as possible.